### PR TITLE
Add maintainers section on readme

### PR DESCRIPTION
**Pull request by Joehuu**
Might be missing some people and roles.

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,20 @@ Home of the [osu!](https://osu.ppy.sh/home) wiki.
 
 Please see [the "contributing" file](CONTRIBUTING.md) if you are interested in helping out with the project!
 
+### Maintainers
+
+Maintainers are users who contribute regularly and establish a solid role. These users are also referred to as wikifriends on the [osu!dev Discord](https://discord.gg/ppy) (`#osu-wiki` channel).
+
+| Name | Tag | Role |
+| :-- | :-- | :-- |
+| [cl8n](https://github.com/cl8n) | `clayton#1349` | Reviewer, code expert |
```
Comment (sr229) [inline README.md]: There's also a couple of people who share this role with him as well, mind you.

Comment (Joehuu) [reply README.md]: I only added people who are consistently active in discord/github.

If you’re talking about `code expert`, that probably needs to be reworded as it is vague atm.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,20 @@ Home of the [osu!](https://osu.ppy.sh/home) wiki.
...
+| [fajar13k](https://github.com/fajar13k) | `fajar13k#1494` | Tournaments maintainer |
+| [Joehuu](https://github.com/Joehuu) | `Joehu#9245` | Reviewer, [ASC](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria) expert |
+| [MegaApplePi](https://github.com/MegaApplePi) | `MegaApple_Pi#9995` | Article maintainer |
+| [TPGPL](https://github.com/TPGPL) | `TPGPL#9098` | Reviewer, pressing butan (merge access) |
```
Comment (TPGPL) [inline README.md]: i think merge access should be enough, as a normal person won't know what pressing butan is about 🤷‍♂

Comment (Joehuu) [reply README.md]: Alright, just wanting to put out the terminology we used.

**Comment by cl8n** [comment]
I don't think this is needed/wanted because people shouldn't have to poke the people listed (except for urgent updates which usually require an admin to refresh anyway..?)

**Comment by Joehuu** [comment]
Alright. Wasn't thinking thoroughly about this. [This page](https://github.com/ppy/osu-wiki/graphs/contributors) is enough for me.

**Pull request closed without merging**